modules/caddyhttp/reverseproxy/reverseproxy.go

```diff
@@ -208,7 +208,13 @@ func (h *Handler) Provision(ctx caddy.Context) error {
 		if err != nil {
 			return err
 		}
-		upstream.networkAddress = addr
+
+		if addr.PortRangeSize() != 1 {
+			return fmt.Errorf("multiple addresses (upstream must map to only one address): %v", addr)
+		}
+    
+    upstream.networkAddress = addr
```

Comment:
```suggestion
	upstream.networkAddress = addr
```
I am the tab police :oncoming_police_car:

Reply:
That said... spaces > tabs :see_no_evil:

Reply:
https://www.youtube.com/watch?v=SsoOG6ZeyUI

Reply:
come on now, I'm not a _monster_... tab key, spaces output.